src/components/user/SignUpForm/index.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { func } from 'prop-types';
+import { Field, reduxForm} from 'redux-form/immutable';
+import { View, Picker } from 'react-native';
+
+import Input from '../../common/Input';
+import MyPicker from '../../common/MyPicker';
+import Button from '../../common/Button';
+import styles, { buttonStyle } from './styles';
+import * as constraints from '../../../utils/constraints';
+
+const parse = value => value === undefined ? undefined : parseInt(value)
```

Comment:
Where do you use this function? 🤔

Reply:
I was trying something but in the end i didn't use it, i'll delete it!